src/main/java/com/goxr3plus/streamplayer/stream/StreamPlayerInterface.java

```diff
@@ -257,6 +257,13 @@
      */
     void setLineBufferSize(int size);
 
+    /**
+     * Set the name of the mixer. This should be called before opening a Line.
```

Comment:
I would add this note to the java doc for the implementation function in Stream Player, as most clients calling the function will be calling it from `StreamPlayer`